english/watari/watari.md

```diff
@@ -0,0 +1,3 @@
+The story of [Watari] (https://www.youtube.com/watch?v=AiTTQ5EW0Ik) the Ninja Boy.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.